app/src/main/res/values/strings_help.xml

```diff
@@ -5,69 +5,42 @@
-        \n- Press the \"clear all\" button in Recent Apps
-        \n- Reboot your smartphone
-        \nAfter one of these actions, the problem should not appear again.
-        \nIf it persists, I am sorry but it seems this will not work for you :-/
-        \n
-        \nTo not be intrusive, the notification icon is transparent and low priority.
-        With some applications having even lower priorities, this may result in an empty space in the status bar.
-        \n
-        \nFinally, if you change the default launcher, you may still see the notification.
-        To hide it, you can open Discreet Launcher (no need to be the default launcher) and disable the notification in the settings.
+        Shows a popup of favorites above other apps when you leave the home screen.
     </string>
     <string name="help_list_update_title">List update</string>
     <string name="help_list_update_text">
-        The list of applications and their icons are refreshed only after some events
-        (on start, when an application is added or removed, if the icon pack is modified).
-        When this happens, a small popup message appears during a few moments.
-        \n
-        \nIf something does not feel right, you should try to refresh the applications list manually.
+        A popup message is shown when refreshing the app list and icons,
+        when starting, adding or removing apps, or if the icon pack is modified.
     </string>
     <string name="help_folders_title">Folders</string>
     <string name="help_folders_text">
-        To create/rename/remove folders and put applications inside, you need to use the folders organizer.
-        It is accessible using the menu button or by clicking on the title of an open folder.
-        \n
-        \nYou can create as many folders as you want to organize your applications.
-        These folders can also be added in favorites.
-        If a folder is removed, its content will be put back in the main applications list.
-        \n
-        \nAn application can be in only one folder at a time (or not in a folder).
-        However, you can have both an application and the folder containing this application in favorites.
+        Manage folders from the 🔧 menu button, or click the title of an open folder
+        to create, rename, favorite it or any app, and/or put an app in it.
+        \nRemoving folders puts all apps in them back in the main list.
     </string>
     <string name="help_search_title">Search</string>
     <string name="help_search_text">
-        The search icon allows to find an application in the complete list.
-        You can add it to favorites in order to access it also using the notification.
-        For technical reasons, it is not possible to add it in a folder.
-        \n
-        \nIf you do not plan to use it, you can also hide the search like any application.
+        Find any app from the search icon. It can also be favorited to be available

```

Comment:
Where is the search, and what does it look like?

Reply:
Its icon is similar to the folder icon but with a magnifier in the center instead of the number of apps contained.
The search was added in v4.1.0 and, unless it has been hidden, you will find it on top of the app drawer.

Until now it was not possible to put the search app in a folder but, following some internal changes, it will become possible in the next release v4.4.0 (not published yet).
So it will really be managed like any other installed app: can be selected in the favorites, put in a folder and hidden from all the views in the launcher.

Reply:
I am on 4.3.0 from F-Droid and I don't have search.
It isn't hidden in "Hidden apps", and I haven't hidden it myself.

Reply:
```suggestion
        Find any app from the 🔍 search icon. It can also be favorited to be available

```